src/peerbook/libp2p_peer.erl

```diff
@@ -332,6 +332,10 @@ decode(Bin) ->
     verify(Msg),
     Msg.
 
+%% @doc Decodes a binary peer without verification, use with care.
+-spec decode_lite(binary()) -> peer().
+decode_lite(Bin) ->
```

Comment:
Could you rename this decode_unsafe? And have the other decode call it before verifying?

Reply:
I will fix this as Evan is in transit.